apps/web/pages/team/[slug].tsx

```diff
@@ -193,7 +193,7 @@ export const getServerSideProps = async (context: GetServerSidePropsContext) =>
     return {
       props: {
         isUnpublished: true,
-        team: unpublishedTeam,
+        team: { ...unpublishedTeam, createdAt: null },
```

Comment:
This was giving an error due to not being able to serialize createdAt property, not used in the page. Proceeded to remove it.